draft-bormann-asdf-instance-information.md

```diff
@@ -231,20 +238,35 @@ TODO: explain how {{RFC9039}} could be used to obtain device names (using `urn:d
 (Describe how protocol bindings can be used to convert these messages
 to/from concrete serializations...)
 
-#### Examples for context information
+TODO: Reference sdfProtocolMap
```

Comment:
Thinking about the protocol map a bit more, I had the impression that it might actually be a good reference point for the Litmus test. Taking the IP address as an example, we could ask ourselves:

- Could the information be retrieved via a protocol from the protocol map?
- Is the information the basis for using the protocol map in the first place?

I guess with something like identity information, there is a gray area where you could model the information both as context information or a property.